doc/source/docker/run-quickstart-examples-docker-compose.rst

```diff
@@ -0,0 +1,114 @@
+Run Flower Quickstart Examples with Docker Compose
+==================================================
+
+Flower provides a set of `quickstart examples <https://github.com/adap/flower/tree/main/examples>`_
+to help you get started with the framework.
+
+These examples are designed to demonstrate the capabilities of Flower and can be run
+using Docker Compose.
+
+.. important::
+
+   Some quickstart examples may have limitations or requirements that prevent them from running
+   on every environment. For more information, please see `Limitations`_.
+
+Prerequisites
+-------------
+
+Before you start, make sure that:
+
+- The ``flwr`` CLI is :doc:`installed <../how-to-install-flower>` locally.
+- The Docker daemon is running.
+- Docker Compose is `installed <https://docs.docker.com/compose/install/>`_.
+
+Run the Quickstart Example
+--------------------------
+
+#. Clone the quickstart example you like to run. For example, ``quickstart-sklearn-tabular``:
+
+   .. code-block:: bash
+
+      $ git clone --depth=1 https://github.com/adap/flower.git \
+           && mv flower/examples/quickstart-sklearn-tabular . \
+           && rm -rf flower && cd quickstart-sklearn-tabular
```

Comment:
```suggestion
           && mv flower/examples/quickstart-pytorch . \
           && rm -rf flower && cd quickstart-pythorc
```

Reply:
i think the pytorch example or the tf would offer a better experience in general